tests/integration/security/mtls_first_party_jwt/strict_test.go

```diff
@@ -35,7 +35,7 @@ func TestMtlsStrictK8sCA(t *testing.T) {
 		Features("security.control-plane.k8s-certs").
 		Run(func(ctx framework.TestContext) {
 
-			rctx := reachability.CreateContext(ctx, p)
+			rctx := reachability.CreateContext(ctx, p, false)
```

Comment:
Add a comment that this won't work with VM and link to the github issue.

Reply:
Working on it! Thanks for reviewing!